example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
```

Comment:
In [OpenWISP](http://openwisp.org) we use [flake8](https://pypi.python.org/pypi/flake8) and it is enforced in the build, but not sure is enforced here. For the moment I suggest you to focus on fixing these small issues I pointed out, it will be easier for you.